src/pages/gen2/build-a-backend/data/data-modeling/identifiers/index.mdx

```diff
@@ -31,12 +31,8 @@ console.log(`New Todo created: ${todo.id}`); // New Todo created: 5DB6B4CC-CD41-
 ```
 
 If you want, you can use Amplify Data to define single-field and composite identifiers:
-- Single-field identifier
-  - With a consumer-provided value (type: `string`)
-  - With an autogenerated value (type: `autoUuid`, `autoUlid`, `autoKsuid`)
-- Composite identifier
-  - With a consumer-provided value (type: `string`)
-  - With an autogenerated value (type: `autoUuid`, `autoUlid`, `autoKsuid`)
+- Single-field identifier with a consumer-provided value (type: `id` or `string`)
+- Composite identifier with a consumer-provided value (type: `id` or `string`)
```

Comment:
I would call out that only required/non-nullable fields can be used as the model identifier